synapse/events/validator.py

```diff
@@ -56,6 +56,17 @@ def validate_new(self, event):
             if not isinstance(getattr(event, s), string_types):
                 raise SynapseError(400, "'%s' not a string type" % (s,))
 
+        if event.type == EventTypes.Aliases:
+            if "aliases" in event.content:
+                for alias in event.content["aliases"]:
+                    if len(alias) > MAX_ALIAS_LENGTH:
+                        raise SynapseError(
+                            400,
+                            ("Can't create aliases longer than"
+                             " %s characters" % MAX_ALIAS_LENGTH),
```

Comment:
for future ref, it's probably preferable to use `%i` for interpolating integers.

It's also preferable to use an explicit tuple for %-interpolation: `"%i characters" % (foo, )`. It's pretty safe here, but if `foo` unexpectedly turned out to be a list or a tuple, bad things would happen, so as a rule: prefer an explicit tuple.